### DISCUSSION TITLE
How to make CC Buttons of the Latched type not send an off message, just on?

### DISCUSSION THREAD

**Discussion by Malensky** [tttapa/Control-Surface#990] [General]
Hello, I have been struggling with this and I feel stuck. My controller needs to send only on toggle messages, not off every second time it is pressed. Do you have an example code for that? Thank you very much in advance.

**Comment by SAEeDGame** [comment]
i was wondering myself its been week but i figure it out
use the example momentary push button  CCButton ,replace the  CCButton in the code with 
CCButtonLatched
